### ISSUE TITLE
Detect decompression commands

### ISSUE THREAD

**Issue by RedXanadu** [coreruleset/coreruleset#2710]
## Description

Part of **JW2SU88A**

Linked PR for this issue: #2712

(PR #2562 also deals with JW2SU88A, but not all of it (or a different aspect of it, perhaps.))

---

We currently catch many Unix and Linux commands, but we don't seem to catch common/obvious use of decompression commands. These could be used to try and exfiltrate data, perhaps from compressed log files, like so:

```
$ gunzip -c /var/log/apache2/access.log.2.gz

->

$ curl --data 'cmd=gunzip -c /var/log/apache2/access.log.2.gz`...
```

It seems like a simple and beneficial fix for us to start catching the most common of these decompression commands.

---

### Update

We _do_ catch **some** decompression commands, but only in the rules that look for a 'starting sequence' that precedes a command in shell syntax, such as:
`; | & $( `` <( >(`

So, we do catch
`;gunzip -c ...`
but not
`gunzip -c ...`

I believe we need to add these commands to the "Unix direct remote command execution" rules, either:
* PL1: 932150
* PL2: 932151

**Comment by RedXanadu** [comment]
Added additional patterns and tests to 932150 and 932160 to catch the described kinds of attacks/patterns. (See #2712 for details.)

**Comment by RedXanadu** [comment]
PR merged following approval.

Closing issue.
